tests/factory_fixtures/ticket.py

```diff
@@ -19,16 +18,12 @@ def _ticket_attributes(issue, tenant, assignedUser, sender):
 
 
 @pytest.fixture
-def create_ticket(
-    faker, ticket_attributes, create_tenant, create_admin_user, create_join_staff
-):
+def create_ticket(faker, ticket_attributes, create_tenant, create_join_staff):
     def _create_ticket(issue=None):
         if not issue:
             issue = faker.sentence()
         tenant = create_tenant()
-        ticket = TicketModel(
-            **ticket_attributes(issue, tenant, create_admin_user(), create_join_staff())
-        )
+        ticket = TicketModel(**ticket_attributes(issue, tenant, create_join_staff()))
```

Comment:
Since we're modifying this lets update it to use the create method instead of bypassing schema validations.